src/backend/executor/process.ts

```diff
@@ -261,6 +261,14 @@ export class ExecutorManager implements Disposable {
         const name = process.processParameters.processType!;
         let namedQueue = this.queue.get(name) ?? [];
 
+        // The exact same task with the exact same commandline won't be added twice
+        for (const queueItem of namedQueue) {
+            if (queueItem.commandLine === process.commandLine) {
+                this.startNextProcess();
+                return;
+            }
+        }
```

Comment:
Let's assume that I would like to see reports in `lib.cpp` file what I modified lately. I save this file, so the analysis will start and let's assume that the analysis takes 1 minutes. Meanwhile I change this source file and save it again. This time the process will be added to the queue and will wait the active process to finish.

The problem is when the active process will be finished the `CodeChecker parse` command will give the following warning message:
```
[WARNING 2022-02-23 08:03] - The following source file contents changed or missing since the latest analysis
- lib.cpp
```
And it is right, the source content is change so the position of the reports are also changed.

For this reason do not we want to kill the current active process if it is the same as the new given process? Similar to this:
```suggestion
        // Kill the active process if it is the same as the given process.
        if (this.activeProcess?.commandLine === process.commandLine) {
            this.killProcess();
        }

        // The exact same task with the exact same commandline won't be added twice
        for (const queueItem of namedQueue) {
            if (queueItem.commandLine === process.commandLine) {
                this.startNextProcess();
                return;
            }
        }
```